libafl/src/executors/hooks/inprocess.rs

```diff
@@ -197,8 +197,7 @@ where
 {
     fn init<E: HasObservers>(&mut self, _state: &mut S) {}
     /// Call before running a target.
-    #[allow(clippy::unused_self)]
```

Comment:
`unused_self` will happen in apple + no_std

Reply:
It does not :)

(At least not on my computer, nor in the CI.)

Reply:
Strange, then the lint changed